pkg/util/vmware/util.go

```diff
@@ -22,8 +23,9 @@ import (
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
-// GetMachineDeploymentName returns the MachineDeployment name for a Cluster.
+// GetMachineDeploymentNameForCluster returns the MachineDeployment name for a Cluster.
 // This is also the name used by VSphereMachineTemplate and KubeadmConfigTemplate.
+// TODO: (killianmuldoon) remove this.
```

Comment:
Nice finding.

Is this TODO for this PR or a follow-up?

Reply:
Follow up - I didn't want to complicate this PR :smile:

Reply:
Removed this to avoid having to rebase